store/tikv/gcworker/gc_worker.go

```diff
@@ -400,14 +406,35 @@ func (w *GCWorker) runGCJob(ctx context.Context, safePoint uint64) {
 		w.done <- errors.Trace(err)
 		return
 	}
-	err = w.doGC(ctx, safePoint)
+
+	useDistributedGC, err := w.checkUseDistributedGC()
 	if err != nil {
-		log.Errorf("[gc worker] %s do GC returns an error %v", w.uuid, errors.ErrorStack(err))
-		w.gcIsRunning = false
-		metrics.GCJobFailureCounter.WithLabelValues("gc").Inc()
+		log.Errorf("[gc worker] %s failed to load gc mode: %v", w.uuid, errors.ErrorStack(err))
```

Comment:
If check gc mode failed, should we fallback to old mode?

Reply:
Emmm, good idea

Reply:
So should the default mode be the old one?